backend/curriculum_tracking/tests/test_RecruitProject.py

```diff
@@ -191,4 +192,44 @@ def test_user_with_team_manage_permissions_can_add_review_on_reviewable_cards(se
     
     def test_user_with_team_manage_permissions_cannot_add_review_on_ip_r_cards(self):
         for card in [self.in_progress_project_card, self.ready_project_card]:
-            self.assertFalse(card.recruit_project.request_user_can_add_review(self.user_with_permissions))
\ No newline at end of file
+            self.assertFalse(card.recruit_project.request_user_can_add_review(self.user_with_permissions))
+
+
+class positive_reviews_since_last_request_review_count_Tests(TestCase):
+    def setUp(self):
+        self.assignee_user = UserFactory()
+        self.competent_reviewers = [UserFactory() for _ in range(2)]
+        self.excellet_reviewer = UserFactory()
+      
+        self.in_review_project_card = factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(
+                content_type=models.ContentItem.PROJECT,
+                project_submission_type=models.ContentItem.LINK,
+            ),
+            status=models.AgileCard.IN_REVIEW,
```

Comment:
This test seems very complicated considering the simplicity of the code under test. As a team, we need to tighten up on our tests.

There seems to be quite a lot of stuff happening in here that is not relevant to what is being tested. Can this be tightened up?

Can't you just:

```
card =  factories.AgileCardFactory(
            status=models.AgileCard.IN_REVIEW,
        )

 factories.RecruitProjectReviewFactory(
                status=COMPETENT,
                recruit_project=card.recruit_project,
                comments="Noice!",
            )

 factories.RecruitProjectReviewFactory(
                status=EXCELLENT,
                recruit_project=card.recruit_project,
                comments="Noice!",
            )
self.assertEqual(card.recruit_project.positive_reviews_since_last_request_review_count,2)
```

Less code is better. The tests should just have code that needs to be there otherwise they end up being hard to read and maintain.

As a team, I think we need to get better at tests. Tests aren't better if they are bigger and more, we need tests to be fast, specific and clear.